include/solana.hpp

```diff
@@ -281,7 +281,7 @@ class Connection {
     json res = json::parse(r.text);
     const std::string encoded = res["result"]["value"]["data"][0];
     const std::string decoded = b64decode(encoded);
-    if (decoded.size() != sizeof(T))
+    if (decoded.size() < sizeof(T))  // decoded should at least fit into T
```

Comment:
better to add a parameter that allows to relax the check, default should be super strict to protect users

Reply:
Makes sense. Reverted that check. If one needs to get subset of the account info, they can specify the `dataSlice` options in #17 : `offset` and `length` to get a smaller part of the account info returned.
i.e
Someone can declare a `MangoAccount` struct with only a subset of the fields and use those `dataSlice` settings to call `getAccountInfo()`.